airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -184,12 +235,12 @@ def execute(self, context: Context):
                     )
                 )
 
-        eks_hook = EksHook(
+        self.eks_hook = EksHook(
             aws_conn_id=self.aws_conn_id,
             region_name=self.region,
         )
```

Comment:
Now that you have a cool base class, you can add an __init__ to it and create the hook inside of that so that all the child classes don't need to create it each time.

Reply:
Nice, that's a good idea. Thanks!

Reply:
Made the changes